src/main/java/chess/service/ChessServiceImpl.java

```diff
@@ -0,0 +1,90 @@
+package chess.service;
+
+import chess.dao.ChessDAO;
+import chess.dao.ChessDto;
+import chess.domain.ChessGame;
+import chess.domain.ChessGameImpl;
+import chess.domain.CurrentGameRoom;
+import chess.domain.PieceForm;
+import chess.domain.Pieces;
+import chess.domain.Position;
+import chess.domain.TeamColor;
+import chess.domain.converter.StringPositionConverter;
+import chess.domain.piece.Piece;
+import chess.exception.InvalidGameException;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional(readOnly = true)
+public class ChessServiceImpl implements ChessService {
+
+    private ChessDAO chessDAO;
```

Comment:
```suggestion
    private final ChessDAO chessDAO;
```
여기는 final을 붙이면 좋을거같아요

Reply:
아하 감사합니다!!